src/index.js

```diff
@@ -66,9 +66,14 @@ class Artifact {
 
   /**
    * Get a URL from the Taskcluster Queue based on task metadata
+   *
+   * If runId is not given, the latest run of the taskId is used.
    */
   async get({taskId, runId, name, filename}) {
-    let url = this.__queue.buildUrl(this.__queue.getArtifact, taskId, runId, name);
+    let url = runId === undefined
```

Comment:
I think this would read more clearly as an if structure, but otherwise looks good to me!

Reply:
Changed.